packages/components/src/components/button/button.tsx

```diff
@@ -86,6 +86,12 @@ export class Button {
    */
   setIconPositionProp() {
     const nodes = Array.from(this.hostElement.childNodes);
+    // delete empty text nodes, which are due to formatting
```

Comment:
I would change "delete" to "skip" or "ignore", because "delete" might give one the impression that it's altering the DOM, when it's only about the array 😅